LSP.sublime-settings

```diff
@@ -6,7 +6,14 @@
       "command": ["pyls"],
       "scopes": ["source.python"],
       "syntaxes": ["Packages/Python/Python.sublime-syntax"],
-      "languageId": "python"
+      "languageId": "python",
+      "environment": {
+        "PYTHONPATH": [
+            "${sublime_libs}",
+            "${packages}",
+            "${project_path}"
+        ]
+      }
```

Comment:
Forgot to remove testing code?

Reply:
Was not sure how to quickly provide an example of how it works. Will remove it.